src/main/java/org/folio/innreach/domain/service/impl/RequestServiceImpl.java

```diff
@@ -210,6 +212,13 @@ private void createOwningSiteItemRequest(InnReachTransaction transaction, User p
     var item = itemService.getItemByHrId(hold.getItemId());
     var requestType = item.getStatus() == AVAILABLE ? PAGE : HOLD;
     var holding = holdingsService.find(item.getHoldingsRecordId()).orElse(null);
+    if (holding != null) {
+      var instance = instanceService.find(holding.getInstanceId()).orElse(null);
+      if (instance != null) {
+        var author = instanceService.getAuthor(instance);
+        hold.setAuthor(author);
+      }
+    }
```

Comment:
`createOwningSiteItemRequest` is also used for Local transactions where this logic is not needed. Move it to `updateTransaction`, add a check for transaction type 

```
if (transaction.getType() == ITEM) {
      
    }
```

Reply:
done